coral/src/app/features/approvals/schemas/components/schema-request-table.tsx

```diff
@@ -0,0 +1,101 @@
+import { Box, DataTableColumn, GhostButton, Icon } from "@aivenio/aquarium";
+import deleteIcon from "@aivenio/aquarium/dist/src/icons/delete";
+import tickCircle from "@aivenio/aquarium/dist/src/icons/tickCircle";
+import infoSign from "@aivenio/aquarium/dist/src/icons/infoSign";
+
+interface SchemaRequestTableData {
+  id: number;
+  topicname: string;
+  environmentName: string;
+  username: string;
+  requesttimestring: string;
+}
+
+const columns: Array<DataTableColumn<SchemaRequestTableData>> = [
+  { type: "text", field: "topicname", headerName: "Topic" },
+  { type: "text", field: "environmentName", headerName: "Cluster" },
+  { type: "text", field: "username", headerName: "Requested by" },
+  {
+    type: "text",
+    field: "requesttimestring",
+    headerName: "Date Requested",
+    formatter: (value) => {
+      return `${value} UTC`;
+    },
+  },
+  {
+    type: "custom",
+    // @TODO PR open in DS to be able to
+    // add an "invisible" header name that
+    // is used as aria-label. That will also
+    // solve the duplicate key warning
+    //https://github.com/aiven/design-system/pull/950
+    headerName: "Details",
+    width: 30,
+    UNSAFE_render: (row) => {
+      return (
+        <GhostButton
+          onClick={() => alert("Approve")}
+          title={`View schema request `}
+          dense
+        >
+          <Box
```

Comment:
I think we cam actually use this approach, where we pass an icon prop to `GhostButton`, instead of doing the layout ourselves in the button :o https://github.com/aiven/klaw/blob/main/coral/src/app/features/approvals/acls/AclApprovals.tsx#L128-L136